src/Bundle/AutoMapperBundle/Tests/ServiceInstantiationTest.php

```diff
@@ -109,6 +109,8 @@ public function testWarmup(): void
     {
         static::bootKernel();
 
-        self::assertFileExists(__DIR__ . '/Resources/var/cache/test/automapper/Symfony_Mapper_Jane_Bundle_AutoMapperBundle_Tests_Fixtures_Address_array.php');
+        self::assertFileExists(__DIR__ . '/Resources/var/cache/test/automapper/Symfony_Mapper_Jane_Bundle_AutoMapperBundle_Tests_Fixtures_NestedObject_array.php');
+        self::assertFileExists(__DIR__ . '/Resources/var/cache/test/automapper/Symfony_Mapper_Jane_Bundle_AutoMapperBundle_Tests_Fixtures_User_array.php');
+        self::assertFileExists(__DIR__ . '/Resources/var/cache/test/automapper/Symfony_Mapper_Jane_Bundle_AutoMapperBundle_Tests_Fixtures_AddressDTO_array.php');
```

Comment:
Could you also test classes are loaded correctly ?

Reply:
done

I had a problem with tests: since we use `AutoMapper::getMapper()` it uses internally `class_exists($className)` to create the php files.

And the files are created in the first test, then classes are loaded, and in the second test, we remove temp directory. Thus at the end the files do exist only in the first test:

```bash
$ vendor/bin/phpunit src/Bundle/AutoMapperBundle/   
PHPUnit 8.5.33 by Sebastian Bergmann and contributors.

^ "class Symfony_Mapper_Jane_Bundle_AutoMapperBundle_Tests_Fixtures_NestedObject_array does NOT exist"
.^ "class Symfony_Mapper_Jane_Bundle_AutoMapperBundle_Tests_Fixtures_NestedObject_array exists"
.^ "class Symfony_Mapper_Jane_Bundle_AutoMapperBundle_Tests_Fixtures_NestedObject_array exists"
.^ "class Symfony_Mapper_Jane_Bundle_AutoMapperBundle_Tests_Fixtures_NestedObject_array exists"
F                                                                4 / 4 (100%)

Time: 1.12 seconds, Memory: 36.00 MB

There was 1 failure:

1) Jane\Bundle\AutoMapperBundle\Tests\ServiceInstantiationTest::testWarmup
Failed asserting that file "/home/nikophil/works/janephp/src/Bundle/AutoMapperBundle/Tests/Resources/var/cache/test/automapper/Symfony_Mapper_Jane_Bundle_AutoMapperBundle_Tests_Fixtures_NestedObject_array.php" exists.
```

so I've moved the `testWarmup()` as the first test, but I don't like this solution.

any idea?

Reply:
I'm fine with this solution if it works, maybe add some comment to explain the issue for the next guy that will have to touch theses tests ? (or just a link to this PR)